plugins/analysis/common/src/main/kotlin/arrow/meta/plugins/analysis/phases/analysis/solver/ast/context/Type.kt

```diff
@@ -20,4 +20,8 @@ interface Type {
   fun isShort(): Boolean
   fun isUnsignedNumberType(): Boolean
   fun isChar(): Boolean
+
+  fun isEqualTo(other: Type): Boolean =
```

Comment:
If this is meant to compare strict equality and not bounds an alternative impl is to leave this abstract and use the underlying kotlin apis for `KotlinType.equals` which delegate to the StrictEqualityTypeChecker
```kotlin
final override fun equals(other: Any?): Boolean {
        if (this === other) return true
        if (other !is KotlinType) return false

        return isMarkedNullable == other.isMarkedNullable && StrictEqualityTypeChecker.strictEqualTypes(unwrap(), other.unwrap())
    }
```

Reply:
Done!